src/assets/i18n/fr.json

```diff
@@ -214,7 +214,7 @@
 	"export csv powerbars description": "Une fois activé, un bouton apparaîtra à côté des Powerbars qui permet d'exporter leurs contenus en tant que fichier CSV",
 	"resultlist title": "Titre du tableau de données",
 	"resultlist title description": "",
-	"Resultlist collection description": "Choisir la collection de données pour cette liste de données",
+	"Resultlist collection description": "ChoiDonnez une unité à chaque collection configurée. Chaque unité apparaîtra à côté du comte correspondant dans l'application d'exploration.ir la collection de données pour cette liste de données",
```

Comment:
**_ChoiDonnez_** une unité à chaque collection configurée. Chaque unité apparaîtra à côté du **_comte_** correspondant dans l'application **_d'exploration.ir_**

=> Donnez une unité à chaque collection configurée. Elle apparaîtra à côté du total d'éléments dans l'application d'exploration

Reply:
It's a copy pase of de l'espace, I don't know it happened. The old value is correct, I fixed it